bot/bot.py

```diff
@@ -22,8 +22,25 @@ def __init__(self) -> None:
 
         self.http_session = ClientSession()
         self.db_pool: asyncpg.Pool = asyncpg.create_pool(constants.DATABASE_URL)
-
-        super().__init__(command_prefix=constants.PREFIX, intents=intents)
+        roles = [
+            Object(r)
+            for r in [
+                constants.Roles.steering_council,
+                constants.Roles.moderators,
+                constants.Roles.gurkult_lords,
+            ]
+        ]
+
+        super().__init__(
+            command_prefix=constants.PREFIX,
+            intents=intents,
+            allowed_mentions=AllowedMentions(
+                everyone=None,
+                users=True,
+                roles=roles,
```

Comment:
```suggestion
        allowed_roles = [
            Object(r)
            for r in [
                constants.Roles.steering_council,
                constants.Roles.moderators,
                constants.Roles.gurkult_lords,
            ]
        ]

        super().__init__(
            command_prefix=constants.PREFIX,
            intents=intents,
            allowed_mentions=AllowedMentions(
                everyone=None,
                users=True,
                roles=allowed_roles,
```
Let's make it explicit.

Reply:
https://github.com/gurkult/gurkbot/pull/158/commits/13db234f55c490ec54c38a658c1dd06f9dbec158